Resources/doc/index.markdown

```diff
@@ -4,7 +4,7 @@
 
 ``` yaml
 "require": {
-    "lunetics/locale-bundle": "2.2.*",
+    "lunetics/locale-bundle": "2.4.*",
```

Comment:
This:
```
"lunetics/locale-bundle": "~2.4",
```
will only update to ``>= 2.4`` stable versions and not to 3.0. It is the same as ``>=2.4 <3.0``. So if this bundle is handling BC breaks correctly (only BC breaks on major versions) it would be a good choice.

Please look at this: https://getcomposer.org/doc/01-basic-usage.md#package-versions